web/scenes/Portal/Teams/TeamId/Team/page/Members/index.tsx

```diff
@@ -48,39 +51,66 @@ export const Members = (props: { teamId: string }) => {
     name: "search",
   });
 
+  const membersRes = useFetchTeamMembersQuery({
+    variables: {
+      teamId,
+      invitesCondition: !search ? {} : [{ email: { _ilike: `%${search}%` } }],
+      membersCondition: !search
+        ? {}
+        : [
+            { user: { name: { _ilike: `%${search}%` } } },
+            { user: { email: { _ilike: `%${search}%` } } },
+          ],
+    },
+  });
+
+  // NOTE: refetch me query to update session in case user role was changed
+  useEffect(() => {
+    if (!membersRes.client || !membersRes.data) {
+      return;
+    }
+
+    membersRes.client.refetchQueries({ include: [FetchMeDocument] });
```

Comment:
@michaelfurutin Why do we need to update it here? You can't update role for yourself.

Reply:
I don’t know why, it was, I was afraid to delete it. I'll try to figure out what was needed

Reply:
we discussed it and decided to leave it